resources/js/Pages/Dashboard/Team/Partials/Worklogs.vue

```diff
@@ -95,16 +95,34 @@
         </p>
 
         <!-- no worklogs yet -->
-        <div v-show="updatedWorklogEntries.length == 0" class="tc mt2">
+        <div v-show="localWorklogEntries.length == 0" class="tc mt2">
```

Comment:
You can (or should?) using `v-if` directive, and also `===`
```suggestion
        <div v-if="localWorklogEntries.length === 0" class="tc mt2">
```